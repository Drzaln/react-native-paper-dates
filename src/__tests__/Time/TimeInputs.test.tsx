import { render } from '@testing-library/react-native'
import TimeInputs from '../../Time/TimeInputs'
import en from '../../translations/en'
import { registerTranslation } from '../../translations/utils'

registerTranslation('en', en)

it('renders TimeInputs', () => {
  const { toJSON } = render(
    <TimeInputs
      inputType="keyboard"
      hours={12}
      minutes={45}
      is24Hour
      locale="en"
      onChange={() => null}
      onFocusInput={() => null}
      focused="hours"
    />
  )

  expect(toJSON()).toMatchSnapshot()
})
