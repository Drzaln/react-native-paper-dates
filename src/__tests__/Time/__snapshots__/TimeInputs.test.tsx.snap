// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders TimeInputs 1`] = `null`;
